cks/values.yaml

```diff
@@ -94,9 +94,23 @@ tolerations: []
 
 affinity: {}
 
+# By default, you can create secrets using the keys below.
+# Important note: Creating secrets in this way is not best practice.
+# While a simpler approach with fewer variables, it is recommended you use
+# a third party secret storage option. Please see README.md for more
+# info.
+
 appSecrets:
   virtruAuth:
     name: hmac-auth
+    data:
+      authTokenJson: <JSON from your CKS>
+      
   virtruKeys:
     name: cks-keys
-    mountPath: /app/keys
\ No newline at end of file
+    mountPath: /app/keys
+    data:
+      rsa001.pub: |
+        rsa001 public key
```

Comment:
maybe use the <> syntax here too for consistency and to indicate that these need to be replaced?